### PR TITLE
Execom moodle theme

### PR DESCRIPTION
**Pull request by fkis1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
-# moodle_customization
-Execom theme applied on Moodle
+# Moodle customization
+Execom theme applied on Moodle
+
+## Setup
+1. Clone repo
+2. Copy repo directory `./moodle_customization/themes/execom` to `path_to_moodle_installation_directory/moodle/theme/` server directory
+3. Visitng the app via browser as admin should automatically initate theme installation on first visit
```
Comment (stanislavgrujic) [inline README.md]: Visitng?

**Comment by stanislavgrujic** [comment]
@iscekic Could you please review CSS?

File: README.md (hunk 1 above)
Comment (fkis1) [reply README.md]: Changed step 3 to: `Login to app as admin. This should automatically initiate theme installation on first visit`.

File: themes/execom/scss/post.scss [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    }
+  }
+}
+
+// Header
+
+.navbar {
+  background-color: $execom-dark-grey !important;
+  color: $white !important;
+  z-index: 1052;
```
Comment (iscekic) [inline themes/execom/scss/post.scss]: Oddly specific. 😄

Comment (fkis1) [reply themes/execom/scss/post.scss]: An empty `.toast-wrapper` div with padding was blocking part of the header (eg. a user couldn't click the Execom logo in the header on smaller screen sizes). As I couldn't reproduce a Toast notification (the core theme possibly doesn't utilize toasts at all), the solution was to set the z-index of the header bar to `1052` as the `.toast-wrapper` had `z-index: 1051`.

I have changed this implementation to hide the padding on the empty `.toast-wrapper` while preserving it's z-index (above the header), so the case where Toast notifications are rendered is covered and will be displayed.

**Pull request merged**